code/modules/mob/living/carbon/brain/posibrain.dm

```diff
@@ -184,3 +191,6 @@
 		var/turf/T = get_turf_or_move(src.loc)
 		for (var/mob/M in viewers(T))
 			M.show_message("\blue The positronic brain pings softly.")
+
+/obj/item/device/mmi/posibrain/ipc
+	silenced = 1
```

Comment:
I can see this ending hilariously badly with people having no idea why the IPC isn't talking
You should probably add this to MMI/examine